stablehlo/reference/Tensor.h

```diff
@@ -67,38 +68,45 @@ class Tensor {
   Tensor();
   explicit Tensor(ShapedType type);
   explicit Tensor(DenseElementsAttr attr);
-  Tensor(const Tensor &other) = default;
+  Tensor(const Tensor &other);
   /// @}
 
   /// Assignment operator.
-  Tensor &operator=(const Tensor &other) = default;
+  Tensor &operator=(const Tensor &other);
 
   /// Returns type of the Tensor object.
   ShapedType getType() const;
 
   /// Returns the number of elements.
   int64_t getNumElements() const;
 
-  /// Provides read access, via a linearized element index, into the
-  /// underlying storage.
-  Element get(int64_t index) const;
+  /// Provides read access to the tensor element indexed at 'indices'.
+  Element get(llvm::ArrayRef<int64_t> indices) const;
 
-  /// Provides write access, via a linearized element index, into the
-  /// underlying storage.
+  /// Provides write access to the tensor element indexed at 'indices'.
   ///
-  /// \param index The index to write to.
+  /// \param indices The multi-dimensional index to write to.
   /// \param element The Element object \a element is used to update the
-  /// underlying storage pointed to by \a index.
-  void set(int64_t index, Element element);
+  /// underlying storage pointed to by \a indices.
+  void set(llvm::ArrayRef<int64_t> indices, const Element &element);
 
   /// Print utilities for Tensor objects.
   void print(raw_ostream &os) const;
 
   /// Print utilities for Tensor objects.
   void dump() const;
 
+  /// Iterator utils.
```

Comment:
Let's expand this comment beyond just "utils", because there's non-trivial semantics involved here. 

Also, calling things "utilities", e.g. `print` and `dump` above, kind of defeats the purpose of having comments for everything. I'm not sure if such comments buy us anything, except for increasing the line of code count :)